submissions/TUstiugov/html-css-popup/index.html

```diff
@@ -0,0 +1,213 @@
+                      </div>
+                      <p class="popup__subtitle">Files</p>
+                    </a>
+                  </li>
+
+                  <li class="popup__list-item">
+                    <a href="#" class="popup__item">
+                      <div class="popup__image">
+                        <img src="./images/popup-icons/contacts.png" alt="">
+                      </div>
+                      <p class="popup__subtitle">Contacts</p>
+                    </a>
+                  </li>
+
+                  <li class="popup__list-item">
+                    <a href="#" class="popup__item">
+                      <div class="popup__image">
+                        <img src="./images/popup-icons/tanslation.png" alt="">
+                      </div>
+                      <p class="popup__subtitle">Translate</p>
+                    </a>
+                  </li>
+
+                  <li class="popup__list-item">
+                    <a href="#" class="popup__item">
+                      <div class="popup__image">
+                        <img src="./images/popup-icons/videos.png" alt="">
+                      </div>
+                      <p class="popup__subtitle">Videos</p>
+                    </a>
+                  </li>
+
+                  <li class="popup__list-item">
+                    <a href="#" class="popup__item">
+                      <div class="popup__image">
+                        <img src="./images/popup-icons/search.png" alt="">
+                      </div>
+                      <p class="popup__subtitle">Search</p>
+                    </a>
+                  </li>
+
+                  <li class="popup__list-item">
+                    <a href="#" class="popup__item">
+                      <div class="popup__image">
+                        <img src="./images/popup-icons/photos.png" alt="">
+                      </div>
+                      <p class="popup__subtitle">Photos</p>
+                    </a>
+                  </li>
+
+                  <li class="popup__list-item">
+                    <a href="#" class="popup__item">
+                      <div class="popup__image">
+                        <img src="./images/popup-icons/apps.png" alt="">
+                      </div>
+                      <p class="popup__subtitle">Apps</p>
+                    </a>
+                  </li>
+
+                  <li class="popup__list-item">
+                    <a href="#" class="popup__item">
+                      <div class="popup__image">
+                        <img src="./images/popup-icons/documents.png" alt="">
+                      </div>
+                      <p class="popup__subtitle">Documents</p>
+                    </a>
+                  </li>
+                </ul>
+                <div class="popup__more">
+                  <input type="checkbox" id="more" class="popup__check">
+                  <label for="more" class="popup__label">More</label>
```

Comment:
Add styles on the focus state to indicate to the user about the current position of focus. 
[scrnli_10_5_2022_8-37-49 PM.webm](https://user-images.githubusercontent.com/16196199/194125622-ba9992c1-c6e9-4d50-a18d-d84d4dc995b6.webm)